docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx

```diff
@@ -1,145 +1,65 @@
 
+class PopupExamplePosition extends React.Component<any, any> {
+  state = { popupOpen: false }
+
+  togglePopupState = () => {
+    this.setState(prev => ({ popupOpen: !prev.popupOpen }))
+  }
+
+  render = () => {
+    const { position, align, icon, padding } = this.props
+
+    return (
+      <Popup
+        open={this.state.popupOpen}
+        align={align}
+        position={position}
+        trigger={
+          <Button
+            onClick={() => this.togglePopupState()}
+            onMouseLeave={() => this.setState({ popupOpen: false })}
+            icon={icon}
+            styles={{ padding, height: '38px', minWidth: '64px' }}
+          />
+        }
+        content={
+          <p>
+            The popup is rendered {position} the trigger<br />aligned to the {align}.
+          </p>
+        }
+      />
+    )
+  }
+}
+
 const triggers = [
-  { icon: 'arrow circle up', padding: '5px 42px 18px 5px' },
-  { icon: 'arrow circle up', padding: '5px 5px 18px 5px' },
-  { icon: 'arrow circle up', padding: '5px 5px 18px 42px' },
-  { icon: 'arrow circle down', padding: '18px 42px 5px 5px' },
-  { icon: 'arrow circle down', padding: '18px 5px 5px 5px' },
-  { icon: 'arrow circle down', padding: '18px 5px 5px 42px' },
-  { icon: 'arrow circle left', padding: '5px 42px 18px 5px' },
-  { icon: 'arrow circle left', padding: '5px 42px 5px 5px' },
-  { icon: 'arrow circle left', padding: '18px 42px 5px 5px' },
-  { icon: 'arrow circle right', padding: '5px 5px 18px 42px' },
-  { icon: 'arrow circle right', padding: '5px 5px 5px 42px' },
-  { icon: 'arrow circle right', padding: '18px 5px 5px 42px' },
-].map(({ icon, padding }) => (
-  <Button icon={icon} styles={{ padding, height: '38px', minWidth: '64px' }} />
-))
+  { position: 'above', align: 'start', icon: 'arrow circle up', padding: '5px 42px 18px 5px' },
+  { position: 'above', align: 'center', icon: 'arrow circle up', padding: '5px 5px 18px 5px' },
+  { position: 'above', align: 'end', icon: 'arrow circle up', padding: '5px 5px 18px 42px' },
+  { position: 'below', align: 'start', icon: 'arrow circle down', padding: '18px 42px 5px 5px' },
+  { position: 'below', align: 'center', icon: 'arrow circle down', padding: '18px 5px 5px 5px' },
+  { position: 'below', align: 'end', icon: 'arrow circle down', padding: '18px 5px 5px 42px' },
+  { position: 'before', align: 'top', icon: 'arrow circle left', padding: '5px 42px 18px 5px' },
+  { position: 'before', align: 'center', icon: 'arrow circle left', padding: '5px 42px 5px 5px' },
+  { position: 'before', align: 'bottom', icon: 'arrow circle left', padding: '18px 42px 5px 5px' },
+  { position: 'after', align: 'top', icon: 'arrow circle right', padding: '5px 5px 18px 42px' },
+  { position: 'after', align: 'center', icon: 'arrow circle right', padding: '5px 5px 5px 42px' },
+  { position: 'after', align: 'bottom', icon: 'arrow circle right', padding: '18px 5px 5px 42px' },
+]
 
 const PopupExamplePosition = () => (
```

Comment:
This const and the class both have the same name which is casing some compiler errors: 
```
[at-loader] Checking finished with 2 errors
[at-loader] ./docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx:4:7
    TS2300: Duplicate identifier 'PopupExamplePosition'.

[at-loader] ./docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx:52:7
    TS2300: Duplicate identifier 'PopupExamplePosition'.
```
Please rename one of them.

Reply:
done 👍